cache/bug_excerpt.go

```diff
@@ -52,6 +54,8 @@ func NewBugExcerpt(b bug.Interface, snap *bug.Snapshot) *BugExcerpt {
 		EditUnixTime:      snap.LastEditUnix(),
 		Status:            snap.Status,
 		Labels:            snap.Labels,
+		Actors:            snap.Actors(),
```

Comment:
You only want to store the id here, not the full identity. The id will need to be resolved in a full Identity later, wherever it's needed (in the query filter most likely).

Reply:
^ this is to be efficient and store the minimal amount of data on disk, as it will be read and loaded in memory each time git-bug start.

Reply:
Okkey I see why the tests failed now, i'll rebase and make the changes